aodndata/soop/soop_xbt_nrt.py

```diff
@@ -366,6 +378,24 @@ def netcdf_writer(profile_data, output_dir):
     template.variables['TEMP_quality_control']['_data'] = np.int8(profile_data['profile_data']['glob_gtspp_temp'])
     template.variables['DEPTH_quality_control']['_data'] = np.int8(profile_data['profile_data']['glob_gtspp_depth'])
 
+    ####################################################################################################################
+    ####################################################################################################################
+    ## In the following part, we modify the qc flags which are GTSPP QC flags and not IODE ones automatically added as
+    ## part of the IMOS netcdf-writer template. We also remove the IMOS part in the conventions gatt since the files
+    ## aren't IMOS compliant anymore. This part may disappear in the future, and the unittests will have to be modified
+    template.global_attributes.update({'Conventions': "CF-1.6"})
```

Comment:
The whole point of templates is that you don't have to do this in code!
All you need to do is update these attribute in your [template file](https://github.com/aodn/python-aodndata/blob/fe3bc678ab91d7af4dbf757109019526bee37024/aodndata/templates/soop_xbt_nrt_bufr_nc_template.json).

Reply:
thanks. completely forgot about that